### PR TITLE
[BUG] fix suggestion list cutoff issue

### PR DESCRIPTION
**Pull request by ananzh**
### Description
In this PR, we fixed the suggestion list cutoff in an easy way. We remove width 250px and add block display to allow browser to select a width and add ellipsis at the bottom.

However, we might have longer input that can't fit into the search. When we truncate the search, the elements look the same which might be still an issue for customer usage. We definitely need more feedbacks on the cutoff behavior.

### Issue resolved:
https://github.com/opensearch-project/OpenSearch-Dashboards/issues/2555

Previous:
Screen Shot 2022-10-18 at 14 45 23

Current:
Screen Shot 2022-10-21 at 10 33 18

 
### Check List
- [x] All tests pass
  - [ ] `yarn test:jest`
  - [ ] `yarn test:jest_integration`
  - [ ] `yarn test:ftr`
- [ ] New functionality includes testing.
- [ ] New functionality has been documented.
- [x] Update [CHANGELOG.md](./../CHANGELOG.md)
- [x] Commits are signed per the DCO using --signoff

### CODE REVIEW

**Comment by ananzh** [comment]
This is a quick fix for the bug to improve current cx usage experience. 
Follow-up issue for suggestion list cutoff behavior is opened here:
https://github.com/opensearch-project/OpenSearch-Dashboards/issues/2606

File: CHANGELOG.md [hunk 1]
```diff
@@ -31,6 +31,7 @@ Inspired from [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 * [Vis Builder] Renames "Histogram" to "Bar" in vis type picker ([2401](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/2401))
 * [MD] Add data source param to low-level search call in Discover ([#2431](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/2431))
 * [Multi DataSource] Skip data source view in index pattern step when pick default ([#2574](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/2574))
+* [BUG] fix suggestion list cutoff issue ([#2607](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/2607))
```
Comment (joshuarrrr) [inline CHANGELOG.md:34]: nit
```suggestion
* [BUG] Fix suggestion list cutoff issue ([#2607](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/2607))
```

**Review by joshuarrrr — DISMISSED** [review]

**Review by ashwin-pc — APPROVED** [review]
From the screenshot, longer strings that are cutoff still look okay to me. Suggestions arent expected to be complete and having a horizontal scrollbar in the suggestion window is not good ux.

**Comment by kavilla** [comment]
> From the screenshot, longer strings that are cutoff still look okay to me. Suggestions arent expected to be complete and having a horizontal scrollbar in the suggestion window is not good ux.

+1, also I wonder if this intentional behavior? Like if hover over and it shows the full string. Is there intention that we can add features here

**Comment by kgcreative** [comment]
> > From the screenshot, longer strings that are cutoff still look okay to me. Suggestions arent expected to be complete and having a horizontal scrollbar in the suggestion window is not good ux.
> 
> +1, also I wonder if this intentional behavior? Like if hover over and it shows the full string. Is there intention that we can add features here

I would +1 that this is an acceptable fix. Honestly, if there are features added, I would expect that they would automatically take up space on the left, so having the width of the container to be automatic with a max-width of 100% still makes sense to me. I think the only item I would add here is that it would be useful to see a truncation elipsis at the end of the cutoff, but that can be explored in a subsequent follow-up. (As far as I know these items are not selectable in their current form, click just loads the string into the search box and then users can select from there, so this feels like a preview that's intended to truncate, and somewhere a max-width got introduced that artificially constrains it.

**Review by kgcreative — COMMENTED** [review]
tl;dr: LGTM 🚀

**Comment by ananzh** [comment]
I changed the fix to allow ellipsis shown at the bottom. Also update commit msg. Thanks for @AMoo-Miki 's idea!

**Review by manasvinibs — APPROVED** [review]
LGTM

**Review by abbyhu2000 — APPROVED** [review]

**Review by AMoo-Miki — APPROVED** [review]
